packages/block-library/src/search/view.js

```diff
@@ -0,0 +1,65 @@
+window.addEventListener( 'DOMContentLoaded', () => {
+	const hiddenClass = 'wp-block-search__searchfield-hidden';
+
+	Array.from(
+		document.getElementsByClassName(
+			'wp-block-search__button-behavior-expand'
+		)
+	).forEach( ( block ) => {
+		const searchField = block.querySelector( '.wp-block-search__input' );
+		const searchButton = block.querySelector( '.wp-block-search__button' );
+		const activeElement = block.ownerDocument.activeElement;
+
+		const ariaLabel = searchButton.getAttribute( 'aria-label' );
+		const id = searchField.getAttribute( 'id' );
+
+		const toggleSearchField = ( showSearchField ) => {
+			if ( showSearchField ) {
+				searchField.removeAttribute( 'aria-hidden' );
+				searchField.removeAttribute( 'tabindex' );
+				searchButton.removeAttribute( 'aria-expanded' );
+				searchButton.removeAttribute( 'aria-controls' );
+				searchButton.removeAttribute( 'aria-label' );
```

Comment:
It's not clear why we're removing `aria-label` here. If the button has a label, it's probably because it's an icon button. In which case, removing the attribute will leave the button nameless. Am I missing something?

Reply:
We remove `aria-label` here since the behavior of the button changes to its default behavior (`type="submit"`). Previously the label would read `Expand search button`. We could set the label to something else, but I thought this would add some unnecessary complexity since the button is performing its expected behavior in this state.

Reply:
I updated this so that aria-label is set to "Submit Search", instead of removing it: [fe7ff74](https://github.com/WordPress/gutenberg/pull/50487/commits/fe7ff74e97fe82fbd0ce3ddc3e57e3d44d2e6646)